content/using_bookwyrm/main-menu.md

```diff
@@ -0,0 +1,123 @@
+This is especially useful if you're in a bookstore or the library!
+
+## Timelines
+
+Timelines are filled with **statuses** from people you **follow** on Bookwyrm.
+It's how you keep up with what your friends are reading, reviewing and recommending. 
+They take up 2/3 of the screen, or on mobile are found below the reading goal section.
+
+### Following other users
+
+As a new user, you will be suggested some accounts to follow.
+You can also search for your friend's username to find and follow them.
+If they are using a different BookWyrm website than you (for example, `bookwyrm.social` and `ramblingreaders.org`)
+then you can still follow them - but you might also need to include their website address in the username, like this:
+
+`user@example.com` or `arthur@ramblingreaders.org`
+
+### Home Timeline
+
+Your Home Timeline is a chronological feed of all the statuses posted by people you follow.
+This includes their reading status updates (want to read, started reading, finished, etc), reviews, ratings, comments, and quotes that they've shared.
+Read more about [types of statuses here](/posting-statuses.html).
+
+Your Home Timeline also includes any statuses that people you follow have **boosted**.
+This is similar to reposting or reblogging on other platforms, and means that status has been shared into your Timeline.
+Boosting is the best way to share good reviews with your followers, and helps them find interesting people to follow.
+This is especially useful since there is no algorithm deciding what to show you,
+only the real humans that you've decided to follow.
+
+### Books Timeline
+
+Your Books Timeline is a chronological feed that collects statuses posted about books that you have in [your shelves](/shelves.html).
+
+It might not display *all* statuses, if your BookWyrm website doesn't know about some
+
+### Interacting with statuses
+
+You can reply to any status in your timeline to start a discussion, and boost or like the status.
+Under 'More Options' there is a link to send a direct message to the poster, report the status, or block them.
+
+### Filtering Timelines
+
+Both timelines have a Filters menu which lets you filter what types of statuses you'd like to see in it.
+You can include or exclude Reviews, Comments, Quotations, or Everything Else (including reading status updates). 
+Then click the 'Apply filters' button to reload the timeline with the new filters.
+
+A little badge called 'Filters are applied' will be shown after filters are changed, 
+but this disappears after another page reload even though filters are still being applied.
+
+## Navigation to other pages
+
+There are many links within the main menu to navigate to other pages on BookWyrm.
+
+### Top menu bar
+
+In the top bar is the search box to go to the search page, and links to [Lists](/lists.html), 
+Discover (recent statuses from the local instance), and [Your Books](/shelves.html).
```

Comment:
Unnecessary new line here will lead to weird text flows depending on screen size.